program_repair.py

```diff
@@ -22,19 +26,46 @@ def replace(input_string, line, column, length, new_string):
     return result
 
 
+def make_mutants(input_string, line):
+    mutants = []
+    length = len(input_string.splitlines()[line - 1]) + 2
+    for i in range(1, length):
+        mutants.append(insert(input_string, line, i, '\''))
```

Comment:
We now insert quotes at all locations from left to right, which is not very effective. For example we generate `pr'int` which we can guess will not work. We can make this more effective by inserting quotes only around spaces present in the erroneous Hedy code?

We can also explicitly disallow inserting quotes inside words.